plugins/commands.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const SCHEMA_COMMAND_NAMESPACE = schema.config.command.namespace;
+const SCHEMA_COMMAND_NAME = schema.config.command.name;
+const SCHEMA_COMMAND_VERSION = schema.config.command.version;
+const DEFAULT_TTL = 24 * 60 * 60 * 1000; // 1 day
+const DEFAULT_BYTES = 1024 * 1024 * 1024; // 1GB
+
+/**
+ * Commands Plugin
+ * @method register
+ * @param  {Hapi}     server                Hapi Server
+ * @param  {Object}   options               Configuration
+ * @param  {Integer}  options.expiresInSec  How long to keep it around
+ * @param  {Integer}  options.maxByteSize   Maximum Bytes to accept
+ * @param  {Function} next                  Function to call when done
+ */
+exports.register = (server, options, next) => {
+    const cache = server.cache({
+        segment: 'commands',
+        expiresIn: parseInt(options.expiresInSec, 10) || DEFAULT_TTL
+    });
+
+    server.expose('stats', cache.stats);
+    server.route([{
+        method: 'GET',
+        path: '/commands/{namespace}/{name}/{version}',
+        config: {
+            description: 'Get command binary',
+            notes: 'Get a script or binary of specific command',
+            tags: ['api', 'commands'],
+            auth: {
+                strategies: ['token'],
+                scope: ['user', 'build']
+            },
+            plugins: {
+                'hapi-swagger': {
+                    security: [{ token: [] }]
+                }
+            },
+            handler: (request, reply) => {
+                const namespace = request.params.namespace;
+                const name = request.params.name;
+                const version = request.params.version;
```

Comment:
Non-blocking comment: we can use ` const { namespace, name, version } = request.params` to make the code cleaner.

Reply:
Fixed to use shorthand.